radixdlt-core/radixdlt/src/main/java/com/radixdlt/environment/Dispatchers.java

```diff
@@ -79,6 +79,7 @@
 /**
  * Helper class to set up environment with dispatched events
  */
+//TODO: get rid of field injection
```

Comment:
Why?

Reply:
- it's inconsistent with other parts of code
- this is generally considered a bad practice

Reply:
added ticket for this issue.